nextcord/application_command.py

```diff
@@ -1914,6 +1915,11 @@ def __init__(
             Permission(s) required to use the command. Inputting ``8`` or ``Permissions(administrator=True)`` for
             example will only allow Administrators to use the command. If set to 0, nobody will be able to use it by
             default. Server owners CAN override the permission requirements.
+        nsfw: :class:`bool`
+            If the command should be usable in NSFW channels or not. Setting to ``True`` will make the command
```

Comment:
```suggestion
            If the command should be solely usable in NSFW channels or not. Setting to ``True`` will make the command
```
Since this implies non age-restricted commands cant be used in restricted channels.